### PR TITLE
bug fix for alter table add multiple columns

### LINKED ISSUE #1025: [BUG]: Transpile - Not able to convert more than 1 add columns in Alter statement #tredence
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Category of Bug / Issue

TranspileParserError

### Current Behavior

**input sql:**

ALTER TABLE remorph_test ADD COLUMN age INT,hire_date DATE;

**Warning Error:**

18:44:53  WARN [sqlglot] 'ALTER TABLE remorph_test ADD COLUMN age INT,hire_date DATE' contains unsupported syntax. Falling back to parsing as a 'Command'.
18:44:53  WARN [sqlglot] 'ALTER TABLE remorph_test ADD COLUMN age INT,hire_date DATE' contains unsupported syntax. Falling back to parsing as a 'Command'.

**in output file:**

Alter statement gets commented out.

### Expected Behavior

ALTER TABLE remorph_test ADD COLUMNS (age INT,hire_date DATE);

### Steps To Reproduce

_No response_

### Relevant log output or Exception details

_No response_

### Sample Query

_No response_

### Operating System

Windows

### Version

latest via Databricks CLI

### PR DESCRIPTION
**Pull request by aman-db**
closes #1025 

Add code for Alter Table Add multiple Columns.
Also added a fix for the default constraint previously it is translated to `GENERATED ALWAYS AS`

### CODE REVIEW

**Review by sundarshankar89 — APPROVED** [review]
LGTM

**Comment by sundarshankar89** [comment]
Fix python format issues LGTM

File: src/databricks/labs/remorph/snow/databricks.py [hunk 1]
```diff
@@ -359,7 +359,7 @@ class Databricks(org_databricks.Databricks):  #
     databricks = org_databricks.Databricks()
     NULL_ORDERING = "nulls_are_small"
 
-    class Generator(org_databricks.Databricks.Generator):
+    class Generator(org_databricks.Databricks.Generator):  # pylint: disable=too-many-public-methods
```
Comment (sundarshankar89) [inline src/databricks/labs/remorph/snow/databricks.py:362]: Move this to the beginning of the file.

@nfx need your override as while overloading the sqlglot functions requires this pylint to be disabled in the file.

Comment (aman-db) [reply src/databricks/labs/remorph/snow/databricks.py]: @sundarshankar89 moved it to the beginning

**Pull request merged**